concord-server/plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Oct 20 08:55:56 NZDT 2016
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-bin.zip
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
Is this the same version we are using for the concord-server ?

Reply:
Yes, needed to upgrade for spotless to work